Resolve plastic molding bottlenecks + add FILE 2 recipe

Tamiore opened the pull request:
#### Summary
Content "Add recipe for metal files, clamps, resolve some other plastic molding bottlenecks"

#### Purpose of change

Right now makeshift vacuum mold require some parts that are either rare or only available if you already _have_ plastic molding equipment. 
Resolves that by adding some new recipes and modifying old ones a bit.

#### Describe the solution
Add a recipe for metal clamp (standard blacksmithing).
Add a recipe for metal files (standard blacksmithing). Not even sure how this one hasn't been added yet...
Add rubber gasket as an alternative to plastic gasket for vacuum pump + recipe for it.
Add a recipe for polycarbonate sheets (basically just flattening down plastic chunks, nothing too fancy).

#### Testing

 .json change, works fine locally.

#### Additional context
Issue cropped up a few times here and there, e.g.: https://www.reddit.com/r/cataclysmdda/comments/p2ezuc/vacuums_have_forsaken_me/

Tamiore commented:
@Maleclypse 
Can you review these and tell me if anything seems off?

Maleclypse commented on data/json/recipes/other/materials.json:
```diff
@@ -126,6 +126,23 @@
+  {
+    "type": "recipe",
+    "activity_level": "fake",
+    "result": "polycarbonate_sheet",
```
I've tried to look up DIY polycarbonate sheets and everything google is bringing me is that you need to buy a polycarbonate sheet extruding machine.   Plus side if we can't find a way to realistically replace them I just increased their spawns several hundred times a couple days ago.  They probably don't need to be homemade anymore.

Maleclypse commented on data/json/recipes/tools/tool.json:
```diff
@@ -955,6 +975,20 @@
+  {
+    "type": "recipe",
+    "activity_level": "fake",
+    "result": "gasket_rubber",
```
I think it would be better to remove vacuum molding requirements from the plastic gasket recipe, than to add an alternative recipe. Also this recipe doesn't currently spawn anywhere.

Maleclypse commented on data/json/recipes/tools/tool.json:
```diff
@@ -955,6 +975,20 @@
+    "skill_used": "fabrication",
+    "difficulty": 2,
+    "time": "20 m",
+    "proficiencies": [ { "proficiency": "prof_plasticworking" } ],
```
Should be a rubber working proficiency.  I imagine they are different.

Maleclypse commented on data/json/recipes/tools/tool.json:
```diff
@@ -955,6 +975,20 @@
+    "time": "20 m",
+    "proficiencies": [ { "proficiency": "prof_plasticworking" } ],
+    "tools": [ [ [ "surface_heat", 5, "LIST" ] ] ],
+    "qualities": [ { "id": "CONTAIN", "level": 1 } ],
```
How does this have contain?   Does the other gasket have contain?  I would think contain is what things that use the gasket would have.

Maleclypse reviewed (changes requested):
While you're here I'd remove the clamps components needed from the makeshift vacuum mold recipe and replace it with the CLAMP tool quality so the clamp isn't consumed and only one is needed.  File and Clamp recipes look good.  Maybe want to make sure the scrap weight is a little more than the weight of the item produced. The new rubber gasket doesn't spawn anywhere currently.  The polycarbonate sheets we'd want to see some kind of evidence or real life recipe on how to make that the way you've suggested. Needs a cited source.   Thanks!

anothersimulacrum commented on data/json/recipes/other/materials.json:
```diff
@@ -126,6 +126,23 @@
   },
+  {
+    "type": "recipe",
+    "activity_level": "fake",
```
Please use a proper activity level here.

Tamiore commented:
As requested, made a number of alterations:
>remove the clamps components needed from the makeshift vacuum mold recipe and replace it with the CLAMP tool quality so the clamp isn't consumed and only one is needed.
Done.


>The polycarbonate sheets we'd want to see some kind of evidence or real life recipe on how to make that the way you've suggested. 
Outside of the scope of this PR. So for now I've just removed this recipe from the PR. 
On a side note:
>I've tried to look up DIY polycarbonate sheets and everything google is bringing me is that you need to buy a polycarbonate sheet extruding machine
How high tech are those "extruding machines" exactly? I mean, we do have pasta extruder. Would an something similar to pasta extruder with build-in heating elements work here?


>I think it would be better to remove vacuum molding requirements from the plastic gasket recipe, than to add an alternative recipe.
Removed rubber gasket and all associated code.
Changed plastic gasket from being vacuum molded to simply being cut from a piece of temperature-soften plastic. (Added basic hammering requirement as a tool to flatten soften plastic into desired diameter before cutting it.)

Maleclypse reviewed (approved):
`How high tech are those "extruding machines" exactly? I mean, we do have pasta extruder. Would something similar to pasta extruder with build-in heating elements work here?`

We'd probably need to find a real book that teaches how to make plastic extruders in your basement.   I'd start looking in forums for DIY 3d printers.

kevingranade commented:
> How high tech are those "extruding machines" exactly? I mean, we do have pasta extruder. Would something similar to pasta extruder with build-in heating elements work here?

IDK about "high tech", but they're likely operating at fairly high pressures and temperatures and require heavy duty machining.
See https://www.weetect.com/polycarbonate-extrusion/ for an overview.

Tamiore commented:
> See https://www.weetect.com/polycarbonate-extrusion/ for an overview.

Look at this source, polycarbonate extrusion actually looks completely feasible to be build at CDDA level of tech (blacksmithing + vehicle-level mechanics).

At any rate, it's not part of _this_ PR any longer. This one seems good to merge from I can tell.
